tpm2_pytss/makecred.py

```diff
@@ -96,3 +97,40 @@ def MakeCredential(public, credential, name):
     credblob = TPM2B_ID_OBJECT(credential=hmacdata + enc_cred)
     secret = TPM2B_ENCRYPTED_SECRET(secret=enc_seed)
     return (credblob, secret)
+
+
+def Wrap(newparent, public, sensitive, symkey, symdef):
```

Comment:
I wish the C code was this clean, looks great and thank you.

Reply:
It's a lot easier when you don't have to throw pointers + sizes around :-)